x-pack/plugin/inference/src/yamlRestTest/resources/rest-api-spec/test/inference/10_semantic_text_field_mapping.yml

```diff
@@ -0,0 +1,105 @@
+setup:
```

Comment:
Could you add some tests for the following cases? Yaml tests give us a ton more coverage.

 - Multi-field (itself as a multi-field and multi-fields UNDER it)
 - nested fields
 - copy to
 - Missing inference chunk text, missing inference results (you already have good tests around bad params or mixed types, huzzah!)

Reply:
the copy-to check will be interesting, we need to be sure to handle it correctly :)